FieldUpdater throws IndexOutOfBoundsException for empty fields

Create a Word document containing just an empty field (w/o any instruction also not containing any spaces):

<img width="105" height="87" alt="Image" src="https://github.com/user-attachments/assets/d2cf2911-1a87-421b-8df4-c31caa051818" />

Word itself does not show an error on field updates.

docx4j throws:

```
...
Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
	at java.base/java.util.Objects.checkIndex(Objects.java:365)
	at java.base/java.util.ArrayList.get(ArrayList.java:428)
	at org.docx4j.model.fields.FieldRef.getFldName(FieldRef.java:296)
	at org.docx4j.model.fields.FieldUpdater.updateComplex(FieldUpdater.java:239)
	at org.docx4j.model.fields.FieldUpdater.updatePart(FieldUpdater.java:93)
	at org.docx4j.model.fields.FieldUpdater.update(FieldUpdater.java:64)
...
```

on `org.docx4j.model.fields.FieldUpdater.update(boolean)`

--- a/docx4j-core/src/main/java/org/docx4j/model/fields/FieldUpdater.java
+++ b/docx4j-core/src/main/java/org/docx4j/model/fields/FieldUpdater.java
@@ -234,6 +234,10 @@ public class FieldUpdater {
 		
 		// Populate
 		for (FieldRef fr : fieldRefs) {
+			if (fr.getInstructions().isEmpty()) {
+				// field w/o instructions (empty)
+				continue;
+			}
 			
 //			if ("DOCPROPERTY".equals(fr.getFldName())) {
 			String fldName = fr.getFldName();

